modules/passkey/contracts/SafeWebAuthnSignerFactory.sol

```diff
@@ -36,6 +36,7 @@ contract SafeWebAuthnSignerFactory is ISafeSignerFactory {
         bytes32 codeHash = keccak256(
             abi.encodePacked(
                 type(SafeWebAuthnSignerProxy).creationCode,
+                "01234567891011121314152546", // Munging
```

Comment:
I think this should be in munging folder, not on the real contract

Reply:
Its not necessary it gets violated anyway